### PR TITLE
#4 import competitor from Thor to new system

### PR DESCRIPTION
**Pull request by shanbel-kassa**

### CODE REVIEW

File: admin/src/athletes/Edit.vue [hunk 1]
```diff
@@ -20,13 +27,25 @@
...
-          :disabled="busy"
-        >
+        <div class="input-group">
+          <input
+            id="kt"
+            v-model="currentItem.kt"
+            type="text"
+            class="form-control"
+            :disabled="busy"
+            @input="event => userId = event.target.value"
```
Comment (bergur) [inline admin/src/athletes/Edit.vue:37]: The value of the input is already bound to currentItem.kt, so you don't have to create a new data binding. You can use currentItem.kt

Comment (shanbel-kassa) [reply admin/src/athletes/Edit.vue]: OK, done.

File: admin/src/athletes/Edit.vue [hunk 2]
```diff
@@ -390,7 +387,7 @@ export default {
           if (res.body.length === 1) {
             this.currentItem = {
               ...res.body[0],
-              newMembership: []
+              newMembership: this.clubs.filter(club => club.thorId === this.currentItem.club)
```
Comment (bergur) [inline admin/src/athletes/Edit.vue:390]: Very nice

Comment (bergur) [reply admin/src/athletes/Edit.vue:390]: Hmm does this work locally for you?
I think `this.currentItem` is not set at this moment so you would have to do: `res.body[0].club)`

Also we need to map it to a membership object. I'm missing the name of the club, the start year, etc.
You can check how a membership array looks like at: https://api.fri.is/athletes/7993

Comment (shanbel-kassa) [reply admin/src/athletes/Edit.vue:390]: oh sorry it is my bad. you are right .we changed to  res.body[0].club.

**Review by bergur — APPROVED** [review]

**Pull request merged**